ccd-definition/CaseEventToFields/RespondExtension.json

```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "RESPOND_EXTENSION",
+    "CaseFieldID": "defendantSolicitor1claimResponseExtensionProposedDeadline",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "Respond",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "RESPOND_EXTENSION",
+    "CaseFieldID": "defendantSolicitor1claimResponseExtensionReason",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "PageID": "Respond",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "RESPOND_EXTENSION",
+    "CaseFieldID": "defendantSolicitor1claimResponseExtensionAccepted",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "MANDATORY",
+    "PageID": "Respond",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "RESPOND_EXTENSION",
+    "CaseFieldID": "defendantSolicitor1claimResponseExtensionCounter",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "Counter",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "defendantSolicitor1claimResponseExtensionAccepted=\"No\"",
+    "ShowSummaryChangeOption": "Y",
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/mid"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "RESPOND_EXTENSION",
+    "CaseFieldID": "defendantSolicitor1claimResponseExtensionCounterDate",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "PageID": "Counter",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "defendantSolicitor1claimResponseExtensionCounter=\"Yes\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "RESPOND_EXTENSION",
+    "CaseFieldID": "defendantSolicitor1claimResponseExtensionRejectionReason",
```

Comment:
Won't this be the claimant's solicitor?

Reply:
In this instance, the thing being rejected is the 'defendantSolicitor1claimResponseExtension'. When multiparty kicks in there will be a RejectionReason attribute for 'defendantSolicitor2claimResponseExtension'. 

We're only expecting a joint rejection from the claimant solicitor(s) and so I've omitted the party doing the rejection from the attribute.